sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/json/SensitivityAnalysisParametersDeserializer.java

```diff
@@ -24,9 +24,9 @@
  *
  * @author Sebastien Murgey <sebastien.murgey at rte-france.com>
  */
-public class SensitivityAnalysisParametersJsonDeserializer extends StdDeserializer<SensitivityAnalysisParameters> {
+public class SensitivityAnalysisParametersDeserializer extends StdDeserializer<SensitivityAnalysisParameters> {
```

Comment:
Not sure it's worth the breaking change to change the name of serializer/deserializer

Reply:
Yes we could simply list that in the breaking changes then, since anyway the PR will bring some breaking changes already.

@phiedw : to your knowledge, will renaming those classes bother "client projects" ?